src/components/menuDropDown/MenuDropDown.js

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+import DropDownItem from "../dropDownItem/DropDownItem";
+import styles from "@/styles/header.module.scss";
+
+const MenuDropDown = ({ itemList }) => {
+  const [showDropDown, setShowDropDown] = useState();
+  return (
+    <ul className={styles.navList}>
+      {itemList.map((item, idx) => {
+        return (
+          <DropDownItem
+            key={Math.random() * 13377331}
```

Comment:
Don't use _random_ keys. React requires keys to be able to identify what elements are the same across renders, e.g. to notice when the order of elements change (and not just consider them completely new elements). Using a random key means they will always be considered new elements every re-render, which is pretty much as bad as not having a key.

If the key is not useful to you, just use the index as the key.